src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -358,34 +360,18 @@ internal static List<ServiceFunction> Parse(string functionNamespace, OpenApiDoc
                     int cacheTimeoutMs = 10000;
                     bool isHidden = false;
                     string description = op.Description ?? $"Invoke {operationName}";
+                    string pageLink = op.PageLink();
 
-                    ServiceFunction sfunc = new ServiceFunction(
-                        null,
-                        theNamespace,
-                        operationName,
-                        operationName,
-                        description, // Template.GetFunctionDescription(funcTemplate.Name),
-                        returnType._type,
-                        BigInteger.Zero,
-                        argMapper.ArityMin,
-                        argMapper.ArityMax,
-                        isBehavior,
-                        isAutoRefreshable,
-                        isDynamic,
-                        isCacheEnabled,
-                        cacheTimeoutMs,
-                        isHidden,
-                        parameterOptions,
-                        argMapper.OptionalParamInfo,
-                        argMapper.RequiredParamInfo,
-                        parameterDefaultValues,
-                        "action", //  funcTemplate.ActionName,??
-                        numberIsFloat,
-                        argMapper._parameterTypes)
+#pragma warning disable SA1117 // parameters should be on same line or all on different lines
```

Comment:
why this change? particularly if we have a style cop rule against it?

Reply:
makes the code unreadable (for me at least,  you might have a different opinion)

we should get rid of this rule + add another one: no more than 10 params to any constructor